CHAID/column.py

```diff
@@ -124,6 +150,10 @@ def groups(self):
     def possible_groupings(self):
         return combinations(self._groupings.keys(), 2)
 
+    def all_combinations(self):
+        bell_set = [ i for i in self.bell_set(self._groupings.keys())]
```

Comment:
This is dumb, I'll make the iterator more optimised whereby we don't have to iterate over it.